nc/include/math/Quaternion.h

```diff
@@ -20,6 +20,8 @@ namespace nc
         static Quaternion FromAxisAngle(const Vector3& axis, float radians); // axis cannot be zero
     };
 
+    Quaternion Normalize(const Quaternion& quat);
```

Comment:
A nit:
This is the only uncommented Quaternion method

Reply:
I put a comment in, but it feels dumb.